lib/src/boringssl/lookup/utils.dart

```diff
@@ -102,7 +102,8 @@ Uri? _findDotDartTool() {
 
   // Find script directory
   Uri root;
-  if (Platform.script.isScheme('data')) {
+  if (Platform.script.isScheme('data') ||
+      Platform.script.toFilePath().startsWith("/tmp")) {
```

Comment:
Is this a new thing?  If this is for detecting that we're running from tests, then is there no better mechanism we can use?